dist/common/scripts/scylla_util.py

```diff
@@ -1008,11 +1008,15 @@ def __load(self):
         self._cfg.optionxform = str
         self._cfg.readfp(f)
 
+    def __fmt(self, val):
+        need_quotes = any([ch.isspace() for ch in val])
+        return '{}="{}"' if need_quotes else '{}={}'
+
     def __escape(self, val):
         return re.sub(r'"', r'\"', val)
 
     def __add(self, key, val):
-        self._data += '{}="{}"\n'.format(key, self.__escape(val))
+        self._data += self.__fmt(val).format(key, self.__escape(val)) + '\n'
```

Comment:
This is weird: why do you both put quotes via `__fmt(...)` but also escape quotes via `__escape(val)`. What is the use case for this?

Reply:
@syuu1228 ping

Reply:
> This is weird: why do you both put quotes via `__fmt(...)` but also escape quotes in the value itself via `__escape(val)`. What is the use case for this?

__escape(val) is just for when val contains double quote in middle of the value , something like `s3.set('ESCAPE1', 'aaaa"bbbb')` I wrote test code below.

> What if the value that arrives is `"disable"`? Don't we want to remove quotes in that case?

It will drop by strip('"') at get(), since strip() function removes both head of the string and tail of the string:
```
    def get(self, key):
        return self._cfg.get('global', key).strip('"')
```
https://github.com/scylladb/scylladb/blob/master/dist/common/scripts/scylla_util.py#L443

Reply:
I see. No more questions. Thanks. ;)